pyOCD/gdbserver/gdbserver.py

```diff
@@ -498,14 +555,14 @@ def resume(self):
         while True:
             if self.shutdown_event.isSet():
                 self.packet_io.interrupt_event.clear()
-                return self.createRSPPacket(val), 0
+                return self.createRSPPacket(val)
 
             # Wait for a ctrl-c to be received.
             if self.packet_io.interrupt_event.wait(0.01):
                 logging.debug("receive CTRL-C")
                 self.packet_io.interrupt_event.clear()
                 self.target.halt()
-                val = self.target.getTResponse(True)
+                val = self.target.getTResponse(forceSignal=signals.SIGINT)
```

Comment:
I get the exception "NameError: global name 'signals' is not defined" when I run this in stop mode